data-raw/save_adlb.R

```diff
@@ -14,5 +19,17 @@ adlb <- read.csv("https://raw.githubusercontent.com/RhoInc/data-library/master/d
                      "Systolic Blood Pressure",
                      "Temperature", "")))
 
+# derive BUN/serum creatinine ratio
+adlb <- adlb_ %>%
+  filter(TEST %in% c("Blood Urea Nitrogen", "Creatinine")) %>%
+  select(-c("STNRLO", "STNRHI")) %>%
+  tidyr::spread(TEST, STRESN) %>%
+  mutate(STRESN = `Blood Urea Nitrogen` / Creatinine,
+         TEST = "Blood Urea Nitrogen/Creatinine",
+         STRESU = "Ratio") %>%
```

Comment:
I'm wodnering if we need to say anything beyond 'ratio', like i could see users asking "ratio of what" for example, how by looking at the chart can you know the units of the creatinine and BUN? Idk what best practice is, may can ask Jim - could be a separate issue we can make if that makes sense to you

Reply:
Agreed - I've opened a new issue #157  and lets ask Jim for best practice.